externals/kyuubi-flink-sql-engine/src/test/scala/org/apache/kyuubi/engine/flink/operation/FlinkOperationSuite.scala

```diff
@@ -163,6 +163,78 @@ class FlinkOperationSuite extends WithFlinkSQLEngine with HiveJDBCTestHelper {
     }
   }
 
+  test("get primary keys") {
+    val tableName1 = "flink_get_primary_keys_operation1"
+    val tableName2 = "flink_get_primary_keys_operation2"
+    val tableName3 = "flink_get_primary_keys_operation3"
+
+    withJdbcStatement(tableName1, tableName2, tableName3) { statement =>
+      statement.execute(
+        s"""
+           | create table $tableName1 (
+           |  id1 int,
+           |  c1 tinyint,
+           |  c2 smallint,
+           |  c3 integer
+           | )
+           | with (
+           |   'connector' = 'filesystem'
+           | )
+    """.stripMargin)
+
+      statement.execute(
+        s"""
+           | create table $tableName2 (
+           |  id1 int,
+           |  id2 int,
+           |  c1 tinyint,
+           |  c2 smallint,
+           |  c3 integer,
+           |  CONSTRAINT pk_con primary key(id1,id2) NOT ENFORCED
+           | )
+           | with (
+           |   'connector' = 'filesystem'
+           | )
+    """.stripMargin)
+
+      statement.execute(
+        s"""
+           | create table $tableName3 (
+           |  id1 int,
+           |  id2 int,
+           |  c1 tinyint,
+           |  c2 smallint,
+           |  c3 integer
+           | )
+           | with (
+           |   'connector' = 'filesystem'
+           | )
+    """.stripMargin)
+
+      val metaData = statement.getConnection.getMetaData
+
+      Seq(tableName1, tableName2, tableName3) foreach { tableName =>
```

Comment:
I found the `tableName3` seems redundant? The definition of `tableName1 ` and `tableName3 ` are almost the same(no primary keys). If we want to verify if we can fetch primary keys, we can just use `tableName1`. WDYT?

Reply:
I think the original idea here is define single pk for `tableName1`

```
CONSTRAINT pk_con primary key(id1) NOT ENFORCED
```